.travis.yml

```diff
@@ -14,7 +14,9 @@ matrix:
 install: pip install -U tox
 
 # command to run tests
-script: tox
+script:
+  - tox -e flake8
```

Comment:
you can put it to matrix, as another env

Reply:
Ok, I managed to do this, it took some time, but... what's the advantage? That moving this to another env will run the flake8 only once, just like a normal test?

Reply:
Yes; also, you won't have all tests failing if that's just a style issue. From job builds it'd be clear if this is a general issue, a problem with flake check, a problem with Python 3.5 only, etc.